src/components/Pages/TeamsPage/OneTeamPage.js

```diff
@@ -591,7 +594,16 @@ const mapStoreToProps = (store) => {
     pref_eq: store.user.pref_equivalence,
   };
 };
-export default connect(mapStoreToProps, null)(withRouter(OneTeamPage));
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators(
+    { toggleGuestAuthDialog: reduxToggleGuestAuthDialog },
+    dispatch
```

Comment:
Can you explain what this is. I'm not literate in redux yet. 😅

Reply:
U see the way you can write something like const `[someState, setSomeState] = useState();` 
And then you use the `setSomeState` function to set the content of `someState` every time in a functional component?

The `reduxToggleGuestAuthDialog` is the `setSomeState` equivalent. And on this part I am trying to register it in the component so it can be recognised as a prop (cos that is the only way you will get to use it 🤣 ). When it has been recognised as a prop, I can retrieve it and fire it anytime inside the component to set the content of some variable inside the redux store, which also triggers an a re-render.... 


--- Yhup, that is kind of the oversimplified explanation. And yeah, redux is not fun at all!! But boy is it necessary 🤣 🤣 🤣 🤣 ....